package.json

```diff
@@ -8,7 +8,7 @@
     "coverage": "nyc --reporter=lcov npm test",
     "lint": "eslint ./lib",
     "release": "standard-version",
-    "sec": "nsp check --output summary",
+    "sec": "npm audit",
```

Comment:
you can get rid of this script. `nsp` was acquired by npm, so you can just run `npm audit` anytime on any project.

Reply:
cool